### DISCUSSION TITLE
new MDN Plus features

### DISCUSSION THREAD

**Discussion by cadeyrn** [mdn/mdn-community#259] [MDN Plus] [answered]
Hi,

thanks for providing such a great documentation for developers! I am subscriber of MDN Plus (MDN Supporter 10), not because I need any of the offered advantages but to support the development of the MDN. 😊

But I wonder what the best way is to get notified about new features. Is there something like a "what's new" page? And is there even a MDN Plus specific roadmap?

Thanks!

**Comment by HerminaC** [comment, marked as answer]
Hi @cadeyrn ! 

First of, thanks for being our supporter! We really appreciate it. 
Roadmap -> Currently we are working on improving the features we've already released: we recently moved from a single collection to multiple, we are adding a notes functionality and we're planning on making collections shareable. The intent here is to be able to follow people you appreciate for their expertise on a certain topic and, by doing this, seeing an enhanced MDN that will not only show you the article content but also that person's note(s) on that content. We hope this will empower our community to learn from one another and share knowledge. 
We're also working on changing the Notifications feature quite drastically, in the sense that we'll go from watching single pages and being notified when changes happen over there, to a notifications feed allowing users to see all notifications at once,  filter them down, and subscribe (to emails, and other push notifications).
As for notifying our users of new functionality, that's a very good question and something we're currently working on - ideally we want to be able to send email notifications but also to add a 'What's new' page directly on the site. 

Hope this helps!
Hermina

**Reply by Ricardo-Dev-Muniz** [reply]
bsmth

**Comment by cadeyrn** [comment]
Hi @HerminaC! Thank you very much for your very detailed answer. 🙂
